### ISSUE TITLE
Settings > Frame no longer available?

### ISSUE THREAD

**Issue by duringleaves** [Swordfish90/cool-retro-term#568]
I recently installed on a new mac system I'm putting together, and was surprised to no longer see options to toggle the display frame.    Didn't see any release notes that this was removed.  Is it a bug on my system?

**Comment by vonHub** [comment]
Seconded--installed on an arch linux VM, no frame rendered and no frame options appeared in settings.

**Comment by Samarang** [comment]
Yes. Nice rough black frame existed in 1.0.1 but the option to select a frame is gone in 1.1.1
Would love to get frames back into the program. MX 19.1

**Comment by kozytskyiad** [comment]
A real shame that this feature is missing, I much prefer the frame I see on old screenshots to the weird blocky screen curvature filler.

**Comment by frankjonen** [comment]
Anything?

**Comment by cyber199** [comment]
For what It's worth, I'm also interested in the return of this feature.
